Lib/unittest/mock.py

```diff
@@ -705,6 +714,11 @@ def __setattr__(self, name, value):
         else:
             if _check_and_set_parent(self, value, name, name):
                 self._mock_children[name] = value
+
+        if self._mock_sealed and not getattr(self, name):
+            mock_name = self._extract_mock_name() + "." + name
```

Comment:
Most of the module use single quotes. It would be nice to follow the existing style.